dev/release/utils-update-docs-versions.py

```diff
@@ -76,14 +85,19 @@
 previous_r_name = old_r_versions[1]["name"].split(" ")[0]
 previous_r_version = ".".join(previous_r_name.split(".")[:2])
 
-if release_type == "major":
+if release_type == "major" and split_version[1:] == ["0", "0"]:
+    # 14.0.0 -> 15.0.0
     new_r_versions = [
         {"name": f"{dev_r_version} (dev)", "version": "dev/"},
         {"name": f"{release_r_version} (release)", "version": ""},
         {"name": previous_r_name, "version": f"{previous_r_version}/"},
         *old_r_versions[2:],
     ]
 else:
+    # 14.0.1 -> 15.0.0
```

Comment:
isn't this case `release_type == "major" and split_version[1:] == ["0", "0"]` true due to release `15.0.0`? I am just confused about the comment.

Reply:
It's false:

* `version = "14.0.1"`
* `split_version = ["14", "0", "1"]` <- `split_version[1:] == ["0", "1"]` not `["0", "0"]`
* `next_version = "15.0.0"`
* `split_next_version = ["15", "0", "0"]`
* `release_type = "major"`